test-files/cmd/slow/ConflateCmdStatsAllDataTypesTest.sh.stdout

```diff
@@ -1,8 +1,8 @@
 stats = (stat) OR (input map 1 stat) (input map 2 stat) (output map stat)
-Node Count	117	116	143	
-Way Count	14	12	18	
+Node Count	117	116	142	
+Way Count	14	12	17	
```

Comment:
Stats look different because there are fewer ways in the output because more were joined.  I believe these stats are more correct than before.